src/ocm/components/clusterDetail/ClusterProperties.tsx

```diff
@@ -37,9 +37,14 @@ const getNetworkType = (
   return networkType;
 };
 
-const getManagementType = (clusterManagementType: boolean | undefined): string => {
+const getAPITitle = (apiName: string, isUserManagementType: boolean | undefined) => {
+  const virtualLabel = isUserManagementType ? '' : ' Virtual';
```

Comment:
```suggestion
  const virtualLabel = isUserManagementType ? '' : ' Virtual';
```
Should depend on whether the cluster is not an SNO instead of the `isUserManagementType`